lib/IRGen/GenDecl.cpp

```diff
@@ -1757,6 +1759,44 @@ void IRGenModule::emitVTableStubs() {
   }
 }
 
+static std::string getEntryPointSection(IRGenModule &IGM) {
```

Comment:
Nit: Since the values are all string constants, you could return `StringRef` from this function (and use it for `sectionName`).

One unnecessary heap allocation is not a big deal, but if you need to edit anything else, you might as well handle this.

Reply:
The signature here matches that of other similar functions in the file, namely `getDynamicReplacementSomeSection` and `getDynamicReplacementSection`, but it may make sense to change them all at some point.